src/Dev-9/FindValueOfKeyInSection.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Class for search value of key in section
+ */
+public class FindValueOfKeyInSection {
+    /**
+     * Find value
+     *
+     * @param A List of lines
+     * @return value of key
+     */
+    public int findValue(String[] A) {
+        DataInput dataInput = new DataInput();
+        int value = 1;
+        if (findSection(A)) {
+            for (int i = 0; i < A.length; i++) {
+                if (A[i].contains(dataInput.getKey())) {
```

Comment:
Here dataInput do not contain key,  the method inputKey was no called, this is bug